static/src/stylesheets/module/_progress.scss

```diff
@@ -0,0 +1,13 @@
+.progress {
+    position: fixed;
+    z-index: 1;
```

Comment:
Is there a z-index variable we can pull in here? I believe we have some defined for each 'plane' on the page.